docs/user/whats-new.asciidoc

```diff
@@ -1,26 +1,190 @@
 [[whats-new]]
-== What's new in {minor-version}
+== What's new in {kib} {minor-version}
+++++
+<titleabbrev>What's new in {minor-version}</titleabbrev>
+++++
 
-This section summarizes the most important changes in this release. For previous
-releases, refer to:
+Here are the highlights of what's new and improved in {minor-version}.
+For detailed information about this release, check out the
+and https://www.elastic.co/blog/kibana-7-8-0-released[release blog], {kibana-ref}/release-notes.html[release notes],
+and {kibana-ref}/breaking-changes.html[breaking changes].
 
-* {kibana-ref-all}/7.7/release-highlights-7.7.0.html[7.7]
-* {kibana-ref-all}/7.6/release-highlights-7.6.0.html[7.6]
-* {kibana-ref-all}/7.5/release-highlights-7.5.0.html[7.5]
-* {kibana-ref-all}/7.4/release-highlights-7.4.0.html[7.4]
-* {kibana-ref-all}/7.3/release-highlights-7.3.0.html[7.3]
-* {kibana-ref-all}/7.2/release-highlights-7.2.0.html[7.2]
-* {kibana-ref-all}/7.1/release-highlights-7.1.0.html[7.1]
-* {kibana-ref-all}/7.0/release-highlights-7.0.0.html[7.0]
-
-For the full list of changes in {minor-version}, see <<release-notes>> and
-<<breaking-changes>>. 
+Other versions: {kibana-ref-all}/7.7/release-highlights-7.7.0.html[7.7] | {kibana-ref-all}/7.6/release-highlights-7.6.0.html[7.6] | {kibana-ref-all}/7.5/release-highlights-7.5.0.html[7.5] |
+{kibana-ref-all}/7.4/release-highlights-7.4.0.html[7.4] | {kibana-ref-all}/7.3/release-highlights-7.3.0.html[7.3] | {kibana-ref-all}/7.2/release-highlights-7.2.0.html[7.2]
+| {kibana-ref-all}/7.1/release-highlights-7.1.0.html[7.1] | {kibana-ref-all}/7.0/release-highlights-7.0.0.html[7.0]
 
 coming[7.8.0]
 
+
+
 //NOTE: The notable-highlights tagged regions are re-used in the
 //Installation and Upgrade Guide
 
 // tag::notable-highlights[]
 
-// end::notable-highlights[]
\ No newline at end of file
+[float]
+[[navigation-7-8]]
+=== Side navigation organized by use cases
+
```

Comment:
How about:

{kib} has a cleaner, more organized side navigation for the Elastic apps. By organizing the apps by use case, you can easily navigate to where you need to go. For example, when you're ready to work with your Logs, Metrics, APM, or Uptime data, you can quickly find where you need to go under *Observability*. The side navigation also includes *Recently viewed* to easily access previously opened apps.